### PR TITLE
fix #412

### LINKED ISSUE #412: non-null field error
This is in the `test_env` image, but it is also reproducable by running `npm run migrate` and `npm run deploy` in the current subgraph repository.

The error is this:
```
{
  "errors": [
    {
      "locations": [
        {
          "column": 11,
          "line": 55
        }
      ],
      "message": "Null value resolved for non-null field `genesisProtocolParams`"
    },
    {
      "locations": [
        {
          "line": 87,
          "column": 11
        }
      ],
      "message": "Null value resolved for non-null field `gpQueue`"
    }
  ]
}
```
And the (query is this:
```
    query ProposalState
          {
            proposals {
              ...ProposalFields
              votes {
                id
              }
              stakes {
                id
              }
            }
          }
          fragment ProposalFields on Proposal {
          id
    
          genesisProtocolParams {
            activationTime
            boostedVotePeriodLimit
            daoBountyConst
            limitExponentValue
            minimumDaoBounty
            preBoostedVotePeriodLimit
            proposingRepReward
            queuedVotePeriodLimit
            queuedVoteRequiredPercentage
            quietEndingPeriod
            thresholdConst
            votersReputationLossRatio
          }
          
          gpQueue {
            id
            threshold
            votingMachine
          }
     
        }
```

### PR DESCRIPTION
**Pull request by ben-kaufman**

### CODE REVIEW

File: ops/mappings.json [hunk 1]
```diff
@@ -534,20 +534,6 @@
...
       },
-      {
-        "name": "UGenericScheme",
-        "contractName": "UGenericScheme",
-        "dao": "base",
-        "mapping": "UGenericScheme",
-        "arcVersion": "0.0.1-rc.33"
-      },
-      {
-        "name": "SchemeRegistrar",
```
Comment (orenyodfat) [inline ops/mappings.json]: why this is removed ?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@daostack/subgraph",
-  "version": "0.0.35-0",
+  "version": "0.0.36-0",
```
Comment (orenyodfat) [inline package.json]: why not make it 0.0.35-1 ?

File: ops/mappings.json (hunk 1 above)
Comment (ben-kaufman) [reply ops/mappings.json]: It isn't. I just moved it down a bit, git showing it weirdly

File: package.json (hunk 2 above)
Comment (ben-kaufman) [reply package.json]: Because I try to keep up with the migration repo which is on v36 already

Comment (orenyodfat) [reply package.json]: not sure it is a good must method . and also I do not see that the migration version is incremented on this pr

**Comment by orenyodfat** [comment]
how this pr fix the issue ?
can we think about a test which catch such issues in the future ?

**Pull request merged**